sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1310,12 +1347,12 @@ export class BlobClient extends StorageClient {
    * operation to copy from another storage account.
    * @see https://docs.microsoft.com/en-us/rest/api/storageservices/copy-blob
    *
-   * @param {string} copySource url to the ource Azure Blob/File.
+   * @param {string} copySource url to the source Azure Blob/File.
    * @param {BlobStartCopyFromURLOptions} [options] Optional options to the Blob Start Copy From URL operation.
    * @returns {Promise<BlobStartCopyFromURLResponse>}
    * @memberof BlobClient
    */
-  public async startCopyFromURL(
+  private async startCopyFromURL(
```

Comment:
>startCopyFromURL [](start = 16, length = 16)

Can customer get raw response from  `beginCopyFromURL` ? Some customers like Yammer likes to get request_id for tracking & logging.

Reply:
@bterlson said this should be made hidden in favor of beginCopyFromURL. I'm not sure if other languages support long-running operations yet but if they do I'll check if they hide `startCopyFromURL`

Reply:
We definitely don't want to expose this functionality in multiple ways. .net has startCopyFromURI, however that returns their new Operation<T> (similar to our poller). We could have done that except we're going with a `begin` convention rather than `start`.

Reply:
Sounds good